spec/views/posts/posts_show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature, js: false do
+  before :each do
+    @user = User.create(name: 'Tom', photo: 'https://randomuser.me/api/portraits/men/2.jpg',
+                        bio: 'Teacher in Mexico')
+    @first_post = Post.create(author: @user, title: 'Hello', text: 'This is my first post')
+    @comment = Comment.create(post: @first_post, author: @user, text: 'Hi Tom!')
+
+    visit "/users/#{@user.id}/posts"
+  end
+
+  describe 'Post page' do
+    it "shows the user's profile" do
+      expect(page).to have_css("img[src*='https://randomuser.me/api/portraits/men/2.jpg']")
+    end
+
+    it "shows the user's username" do
+      user_name = Post.find_by(author: @user.name)
+      expect(page).to have_content(user_name)
+    end
```

Comment:
- [ ] @Team kindly change this test from testing if the username is seen to the author's username is seen as per the project requirement
![image](https://github.com/Patfarmurs/myblog/assets/61408860/4e243273-da06-43b9-a501-4b34ff5ddc70)

- [ ] @Team you also forgot to include the tests below, kindly consider adding them.
 

- I can see the username of each commentor.
- I can see the comment each commentor left.

![image](https://github.com/Patfarmurs/myblog/assets/61408860/88c387ff-7d7f-42ff-b2a6-809d4ce65f3b)